package.json

```diff
@@ -1,42 +1,48 @@
 {
-  "name": "nemo-screenshot",
```

Comment:
Yeah, if you don't mind. I think this PR is morphing into essentially a set of whitespace changes.